Source/Tools/FEXLoader/WindowsDummyHandlers.cpp

```diff
@@ -0,0 +1,27 @@
+#ifdef _WIN32
+#include "WindowsDummyHandlers.h"
```

Comment:
Did you miss adding this file?

Reply:
Yes I did. I thought this was already there but I had missed it on my first commit of mingw support.